website/frontend/templates/downloads.html

```diff
@@ -141,6 +141,12 @@
                 </thead>
                 <tbody>
                   <tr>
+                    <td>{{ _("All") }} (Flatpak)</td>
```

Comment:
The `_("All")` should probably have some kind of qualifier. Either changing the string itself ("All distributions" or "All Linux distributions") or using a gettext context. I'm not sure whether gettext contexts are implemented in the Picard website code yet though.